live_site.py

```diff
@@ -0,0 +1,433 @@
+
+	for each_droplet, details in enumerate(my_droplets['droplets']):
+		for key, value in details.items():
+			x = details['name']
+			dropletId = details['id']
+			return dropletId
+	else:
+		return None		
+
+
+def get_hostname_ip(droplet):
+# This function is to cycle through all the droplets and 
+# get the IP Address of the droplet.
+# Also checks if the droplet is up, thus returns the IP Address.  
+	result = ''
+	ip = ''
+
+	click.secho(">>> Inspecting droplets...", fg='bright_yellow')   
+	my_droplets = get_all_droplets()	
+	
+	for each_droplet, details in enumerate(my_droplets['droplets']):
+		for key, value in details.items():
+			if details['name'] == droplet:
+				#click.echo(details['name'])                
+				if key == 'networks':
+					result = value      
+					for key, value in result.items():
+						if key == 'v4':
+							v4 = value[1]
+							ip = v4['ip_address']
+							#click.echo(ip)
+							return ip
+	else:
+		click.secho(">>> Your droplet {}' doesn't exist!".format(droplet), fg='bright_red')
+		click.secho(">>> We can't add your data at the moment.", fg='bright_red')                     
+
+
+def get_all_domains():
+
+	api_url = '{0}domains'.format(api_url_base)
+
+	response = requests.get(api_url, headers=headers)
+
+	if response.status_code == 200:
+		return json.loads(response.content.decode('utf-8'))
+	else:
+		return None
+
+
+def add_new_domain(domain_name, ip_address):
+
+	api_url = '{0}domains'.format(api_url_base)    
+
+	domain = {'name': domain_name, 'ip_address': ip_address}
+
+	response = requests.post(api_url, headers=headers, json=domain)
+
+	if response.status_code == 201:
+		domain = json.loads(response.content)
+		return domain
+	else:
+		click.secho('[?] Unexpected Error: [HTTP {0}]: Content: {1}'.format(response.status_code, response.content), fg='bright_red')
+		return None
+			
+
+def check_domain(domain_name):
+
+	my_domains = get_all_domains()
+	domain_exists = ''
+
+	# Checks if the domain name already exists or not
+	for domain, details in enumerate(my_domains['domains']):
+		for key, value in details.items():
+			x = details['name']            
+			if domain_name != x: continue
+			domain_exists = x
+			return domain_exists
+		else:
+			return None	
+	else:
+		return None
+
+
+def will_add_domain(droplet, domain_name):
+		# IP address is needed to create the domain name, so we call the function get_hostname_ip()
+		ip = get_hostname_ip(droplet)
+		# Just want to make sure the ip_address is a string, so I convert it anyway.
+		ip_address = str(ip)  
+
+		# This adds/creates the domain name
+		new_domain = add_new_domain(domain_name, ip_address)
+
+		if new_domain is not None:
+			click.secho('>>> Your new domain name {} has been created with IP Address {}. '.format(domain_name, ip_address), fg='bright_green')
+		else:
+			click.secho('[!] Request Failed', fg='bright_red')
+
+
+def create_domain(droplet, domain_name):
+
+	domain_exists = check_domain(domain_name)
```

Comment:
One thing I have to say about this variable name - I can see what you're doing, you're getting the domain name, and if it's blank (or somehow just doesn't match the expected domain_name) then you know that the domain exists. But when I see a variable like `domain_exists` my instinct as a programmer is to assume it is a boolean. If you want it to contain a domain registration, then call it that - but domain_exists sounds like a boolean